README.md

```diff
@@ -7,6 +7,30 @@ or dynamic objects, and articulated robots with various sensors, and acutators.
 The format of SDFormat is also described by XML, which facilitates updates and
 allows conversion from previous versions.
 
+## Documentation
+
+See the [SDFormat Website](http://sdformat.org/) for:
+
+* [An outline of the SDFormat XML specification](http://sdformat.org/spec)
+* [Documentation on specification specifics](http://sdformat.org/tutorials?cat=specification)
+* [Proposals for modifications](http://sdformat.org/tutorials?cat=pose_semantics_docs)
+* [Developer instructions](http://sdformat.org/tutorials?tut=roadmap)
+  (e.g. installation and a roadmap)
```

Comment:
That page is a bit misleading to me, when I go to a roadmap, I'd expect to see a list of features planned for future releases. That page also doesn't have installation instructions, only version information. I think it should be called `releases`, similar to this one: https://ignitionrobotics.org/docs/all/releases

Reply:
Aye, will remove it from this for now, and create an issue on `sdf_tutorials`.

Reply:
Done. Consolidated to namely direct towards